src/armsgs.py

```diff
@@ -20,8 +20,9 @@ def __init__(self, log, debug, last_updated, version, verbose, colors=True):
         ----------
         log : str
           Name of saved log file (no log will be saved if log=="")
-        debug : bool
-          Used for debugging. Should be set to False in all other cases
+        debug : dict
```

Comment:
On line 124, there is a reference to self._debug that needs to be replaced with:
if self._debug['develop']: